guides/v2.1/ui_comp_guide/concepts/ui_comp_uiregistry.md

```diff
@@ -24,13 +24,33 @@ Now we have `uiRegistry` instance in the `registry` variable. We can use it to g
 var component = registry.get('%componentName%');
 ```
 
-For example:
+The `uiRegistry` instance allows to search the component by some property.
+If you know some unique property of the component that you need to find, you can use the following code to get this component:
+
+```javascript
+var component = registry.get('%property% = %propertyValue%');
+```
+
+**Example**
+
+Example with getting component by full component name:
```

Comment:
Please, rephrase as "Getting a component by the full name:" (remove "Example with"). This section is already wrapped by "Example" title.